vtexdocs/devportal
# Feature/api guides see also

#### What is the purpose of this pull request?

Add the see-also-section component to the api-guides documentation pages.
mode details [here.](https://www.notion.so/vtexhandbook/Adicionar-se-o-See-also-em-p-ginas-de-API-Guides-662e8873c5534a5dbb8c75a25142532c)
#### What problem is this solving?

The api-guides documentation pages should have a "see also" section, which was absent. To create the "see-also-section" component,  the previously known "dropdown-card" was adapted to fit the "see-also-section" as well as the "dropdown-menu" component.
#### How should this be manually tested?

First, check the tests ran by the GitHub actions. Then, go to [this page](https://deploy-preview-43--elated-hoover-5c29bf.netlify.app/), try different screen resolutions and check if it follows the designs in [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2351%3A133100).
#### Screenshots or example usage

#### Types of changes
